src/llvm_bindings.zig

```diff
@@ -0,0 +1,507 @@
+//! We do this instead of @cImport because the self-hosted compiler is easier
+//! to bootstrap if it does not depend on translate-c.
+
+const std = @import("std");
+const assert = std.debug.assert;
+
+const LLVMBool = bool;
```

Comment:
Not sure if this is intentionally different, but the LLVM C API's bool is a typedef to `int` as far as I can tell.